vespa-feed-client/src/main/java/ai/vespa/feed/client/JsonFeeder.java

```diff
@@ -284,6 +309,21 @@ String getDocumentJson(long start, long end) {
         }
     }
 
+    private class SingleOperationParserAndExecutor extends OperationParserAndExecutor {
+
+        private final byte[] json;
+
+        SingleOperationParserAndExecutor(byte[] json) throws IOException {
+            super(factory.createParser(json), false);
+            this.json = json;
+        }
+
+        @Override
+        String getDocumentJson(long start, long end) {
```

Comment:
We should probably just use `byte[]` as data carrier, all the way through to the HTTP layer?

Reply:
Not very related to this PR, though :)